src/browser/Sync.js

```diff
@@ -0,0 +1,20 @@
+function notSupported() {
+    console.log('ContentSync is not supported on browser platform');
+}
+
+var ContentSync = function() {};
+ContentSync.prototype.on = function() {};
+ContentSync.prototype.emit = function() {};
+ContentSync.prototype.cancel = function() {};
```

Comment:
Is there a reason why the `.on`, `.emit`, and `.cancel` do not use the `notSupported()` function as well?